### DISCUSSION TITLE
How to correctly downgrade Postal?

### DISCUSSION THREAD

**Discussion by 007hacky007** [postalserver/postal#2474] [Help with using Postal] [answered]
I'm currently experiencing some issues with the latest version ( https://github.com/postalserver/postal/issues/2466 ) and so I would like to downgrade to back to the [2.1.2](https://github.com/postalserver/postal/releases/tag/2.1.2). 

How to properly execute the downgrade and not lose existing data and configuration.

Thank you in advance for your help!

**Comment by willpower232** [comment, marked as answer]
You should be able to edit the docker-compose.yml file to change the version number.

https://github.com/postalserver/install/blob/main/templates/docker-compose.yml

I would suggest running `postal stop` first so you can then start it with the version number you want.

**Reply by 007hacky007** [reply]
Thank you! 

1. `postal stop`
2. Changed version numbers in `/opt/postal/install/docker-compose.yml` 
3. `postal start`
4. running with the 2.1.2 version again!
